src/lib/pages/query/components/QueryArea.tsx

```diff
@@ -89,17 +89,17 @@ export const QueryArea = ({
 
   return (
     <Flex direction="column">
-      <Box width="full" my="16px" alignItems="center">
+      <Box width="full" mt="4" mb="8" alignItems="center">
         {contractAddress && (
-          <Text variant="body3" mb="8px">
+          <Text variant="body3" mb="2">
```

Comment:
I prefer to write mb={2} to differentiate between arbitrary unit ("8px") and chakra space format (2)